admin/views/add_subscription_plan_view.php

```diff
@@ -38,7 +38,7 @@
 							<tr valign="top">
                                 <th scope="row">Recurring Amount:</th>
                                 <td>
-                                    <input type="text" name="plan_recurring_amount" min="10" required
+                                    <input type="text" name="plan_recurring_amount" min="0" required
                                            value="<?php echo $plan_row ? number_format($plan_row->subscription_plan_amount, 2) : ''; ?>"/>
```

Comment:
@Oj-Michael for this change: 
1) What happens when a subscription plan is free/trial, does the checkout still redirect to BananaCrystal payme page and fill in the amount as USD 0.00? Is that necessary?

2) Also, will a transaction with $0.00 go through? There is validation on the backend that will  most likely require a minimum amount > 0

It would also be helpful to include a video with the $0.00 subscription case.

Reply:
> @Oj-Michael for this change:
> 
> 1. What happens when a subscription plan is free/trial, does the checkout still redirect to BananaCrystal payme page and fill in the amount as USD 0.00? Is that necessary?
> 2. Also, will a transaction with $0.00 go through? There is validation on the backend that will  most likely require a minimum amount > 0
> 
> It would also be helpful to include a video with the $0.00 subscription case.

Hello ma, I was not aware of the business logic for the subscription plan. Modifiy this code was not in the scope and I'm not sure what happens during a subscription... I made this change because not all the stores will be USD stores; sorry about it and I will revert.